.github/workflows/ci.yml

```diff
@@ -1,12 +1,14 @@
-name: OpenLibm CI
-
+name: CI
 on:
-  push:
   pull_request:
-    branches: [ master ]
-
+    branches:
+      - master
+  push:
+    branches:
+      - master
```

Comment:
sorry for taking that out if you wanted it.  It made my testing harder on my fork because I wasn't using the 'master' branch there

Reply:
No worries!